siliconcompiler/tools/openroad/openroad.py

```diff
@@ -68,7 +70,14 @@ def setup(chip, mode='batch'):
     chip.set('eda', tool, 'option',  step, index, option, clobber=clobber)
     chip.set('eda', tool, 'refdir',  step, index, refdir, clobber=clobber)
     chip.set('eda', tool, 'script',  step, index, refdir + script, clobber=clobber)
-    chip.set('eda', tool, 'threads', step, index, os.cpu_count(), clobber=clobber)
+
+    # normalizing thread count based on parallelism and local
+    threads = os.cpu_count()
+    if not chip.get('remote') and step in chip.getkeys('flowgraph', flow):
```

Comment:
```suggestion
    if (not chip.get('remote')) and (step in chip.getkeys('flowgraph', flow)):
```

Probably not necessary, but C has left me with a habit of using parentheses to over-define the order of operations.